static/scripts/audit-report/helpers.ts

```diff
@@ -177,10 +177,10 @@ export function isValidUrl(urlString: string) {
   }
 }
 
-export function getCurrency(comment: string) {
-  if (comment.includes("WXDAI")) {
+export function getCurrency(id: number) {
+  if (id === 100) {
```

Comment:
This should be handled dynamically I think. We should be able to support any network and any token.

Reply:
> no clue if this value impacts any other project

I don't understand what you're saying but network 100 is Gnosis Chain which we use for most of our payouts now. However we want partners to be able to use any network (just as long as there is a [permit2 smart contract deployment on it](https://blockscan.com/address/0x000000000022d473030f116ddee9f6b43ac78ba3), it should be fine!)